evap/static/scss/components/_notebook.scss

```diff
@@ -43,42 +43,43 @@
         flex-grow: 1;
     }
 
-    @media screen and (min-aspect-ratio: $notebook-break) {
-        .notebook-container {
-            width: 25vw;
-            padding-left: $spacer;
-            padding-bottom: $spacer;
-            padding-top: $spacer;
-        }
+    display: flex;
+    justify-content: center;
+    .notebook-container {
+        width: 80vw; //approximation to be similar to evap-Content width
+        z-index: $zindex-fixed;
```

Comment:
We only change sizes, if I understand correctly. Why is a change to z-index necessary?